src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import { TodoForm } from '../TodoForm/TodoForm';
+
+import cn from 'classnames';
+import { useTodos } from '../context/TodosContext';
+
+export const TodoHeader = () => {
+  const { todos, toggleAll } = useTodos();
+  const allCompleted = todos.every(todo => todo.completed);
```

Comment:
```suggestion
  const isAllCompleted = todos.every(todo => todo.completed);
```

let's use `is/has` prefix for boolean